lib/bundle/brew_dumper.rb

```diff
@@ -61,6 +56,18 @@ def expand_cask_requirements
 
     private
 
+    def self.formulae_info
+      @@formulae_info ||= begin
```

Comment:
We're using class variables elsewhere so I thought it was better to just be consistent here. Caching after sorted makes sense though :+1: